SU2_CFD/src/solvers/CEulerSolver.cpp

```diff
@@ -6502,6 +6507,9 @@ void CEulerSolver::BC_Giles(CGeometry *geometry, CSolver **solver_container, CNu
           Jacobian.SubtractBlock2Diag(iPoint, residual.jacobian_i);
 
       }
+      /*--- Store number of Newton iterations at BC ---*/
+      if(config->GetKind_FluidModel() == DATADRIVEN_FLUID)
+        GetNodes()->SetNewtonSolverIterations(iPoint, GetFluidModel()->GetnIter_Newton());
```

Comment:
Inside the solvers you use "nodes" directly
```suggestion
        nodes->SetNewtonSolverIterations(iPoint, GetFluidModel()->GetnIter_Newton());
```